Model/QueuedTask.php

```diff
@@ -18,11 +18,11 @@ class QueuedTask extends AppModel {
  * Adds a new Job to the queue.
  *
  * @param string $taskName A queue task name
- * @param mixed $data Any data
+ * @param array $data Any data
  * @param string $notBefore A datetime which indicates when the job may be executed
  * @return mixed On success `Model::$data` if its not empty or true, false on failure
  */
-	public function createJob($taskName, $data, $notBefore = null) {
+	public function createJob($taskName, array $data, $notBefore = null) {
```

Comment:
want to add additional type hints?

Reply:
Yes, but I'm not about the last one

```
public function createJob(string $taskName, array $data, ?string $notBefore = null) {
```

?

Reply:
`?string` does not work in PHP 7.0. I did not add any hints for `$notBefore` for now